iothub/device/src/ConnectionStatusChangeReason.cs

```diff
@@ -8,13 +8,59 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public enum ConnectionStatusChangeReason
     {
+        /// <summary>
+        /// This signifies that the client is connected, and ready to be used.
+        /// <para>This connection status change reason is returned with a connection status of <see cref="ConnectionStatus.Connected"/>.</para>
```

Comment:
What does this mean? I don't think I followed. Might want to revisit this description for users that have not used IotHub a lot.

Reply:
which part of the description is unclear?

Reply:
This I did not get - This connection status change reason is returned with a connection status of <see cref="ConnectionStatus.Connected"/>.</para>

Reply:
The client will receive the connection status change reason as a part of the connection status change callback, the signature of which is:
```csharp
public delegate void ConnectionStatusChangesHandler(ConnectionStatus status, ConnectionStatusChangeReason reason);
```
So, the connection status change reason is always returned with an associated connection status.